sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -1135,6 +1126,11 @@ public partial class JsonObjectSerializer : Azure.Core.Serialization.ObjectSeria
         public override System.Threading.Tasks.ValueTask SerializeAsync(System.IO.Stream stream, object? value, System.Type inputType, System.Threading.CancellationToken cancellationToken) { throw null; }
         public override System.Threading.Tasks.ValueTask<System.BinaryData> SerializeAsync(object? value, System.Type? inputType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    public enum NameConversion
```

Comment:
I worry this might be a little too general - even for Azure.Core.

Reply:
The goal here was to keep it short, and to be consistent in naming with `IMemberNameConverter` (the Azure.Core.Serialization interface).